modules/phase_field/tests/phase_field_kernels/tests

```diff
@@ -60,4 +60,11 @@
     input = 'ConstructionOrder.i'
     exodiff = 'ConstructionOrder_out.e'
   [../]
+
+  [./SimpleSplitCHWRes]
+    type = 'Exodiff'
+    input = 'SimpleSplitCHWRes.i'
+    exodiff = 'SimpleSplitCHWRes_out.e'
+    max_parallel = 1                
```

Comment:
I followed the block of SplitCahnHilliard. I don't think it's really useful, but not sure about it. So I just keep it there.